examples/text-generation/utils.py

```diff
@@ -557,6 +557,12 @@ def setup_tokenizer(args, model, assistant_model):
         tokenizer.eos_token = tokenizer.decode(tokenizer.eos_token_id)
         tokenizer.bos_token = tokenizer.decode(tokenizer.bos_token_id)
 
+    # HACK: MiniCPM3 has multiple eos_tokens and does not specify padding token. Set both to second one.
+    if model.config.model_type == "minicpm3":
+        tokenizer.pad_token = tokenizer.eos_token
+        model.generation_config.pad_token_id = model.generation_config.eos_token_id[-1]
+        model.generation_config.eos_token_id = model.generation_config.eos_token_id[-1]
```

Comment:
Can we have a warning printed that we are only gonna use 1 eos, eos_token_id[-1], somtthing like that. that way user is not confused why it didnt stop even though some eos got generated.

Reply:
Added a print warning to this.